great_expectations/data_context/store/expectations_store.py

```diff
@@ -210,9 +210,14 @@ def _add(self, key, value, **kwargs):
             if self.cloud_mode:
                 # cloud backend has added IDs, so we update our local state to be in sync
                 result = cast(GXCloudResourceRef, result)
-                cloud_suite = ExpectationSuite(
-                    **result.response["data"]["attributes"]["suite"]
-                )
+
+                suite_kwargs = result.response["data"]["attributes"]["suite"]
+
+                # Temporary fork to account for pre-V1 configs
+                if "ge_cloud_id" in suite_kwargs and "id" not in suite_kwargs:
+                    suite_kwargs["id"] = suite_kwargs.pop("ge_cloud_id")
```

Comment:
Added this to get E2E tests passing - should we just xfail instead?

Reply:
Is the e3e test interacting with cloud or with file-based context? if cloud, I'd say what you have here is good, and we should just create a small ticket to track cleaning this up.

Reply:
i think this is worth it to maintain test coverage